src/engraving/libmscore/chord.cpp

```diff
@@ -3746,18 +3746,19 @@ void Chord::layoutArticulations()
         bool headSide = bottom == up();
         qreal x = centerX();
         qreal y = 0.0;
-
+        const qreal halfBeamSp = score()->styleS(Sid::beamWidth).val() * _spatium() * 0.5;
         if (bottom) {
             if (!headSide && stem()) {
+                auto userLen = stem()->userLength();
                 if (_up) {
-                    y = downPos() - stem()->length();
+                    y = downPos() - stem()->length() - userLen;
                     if (beam()) {
-                        y -= score()->styleS(Sid::beamWidth).val() * _spatium * .5;
+                        y -= halfBeamSp * beam()->mag();
```

Comment:
Doesn't `_spatium` (as part ot `halfBeamSp`) already contain `mag`?

Reply:
`beam->mag()` can be different from `mag` (which is based on the magnitude of the chord) in situations like this
![image](https://user-images.githubusercontent.com/89263931/156016076-b40c3661-abeb-487b-9993-63293f5b1b74.png)
... as far as I can tell. So instead of using `_spatium` for that one I'll use `score()->spatium()` so that the chord magnitude isn't included in the calculation

Reply:
OK, still my last question: why not doing this only once?

Reply:
i guess we could, but `beam()` is not guaranteed to be non-null until the check, so we'd need to add another check where the definition is. I don't see it as too big of a deal though